environments/gcp/flyte-core/services.tf

```
@@ -0,0 +1,14 @@
+locals {
+  enabled_services = [
+    "servicenetworking.googleapis.com", 
+    "container.googleapis.com",
+    "compute.googleapis.com"
```

```suggestion
    "compute.googleapis.com",
    "sqladmin.googleapis.com"
```

I'm using Pulumi, not Terraform. So Database creation is slightly different and this service should be enabled. Or maybe I'm wrong =) anyway, that helped me.

---

@NikitaSemenovAiforia Interesting, are you operating the DB with a Service Account? That would be better than local PG users (as it's done here) and would explain why you need the SQL Admin API.

I've been circulating the idea to build a ref implementation with the Pulumi Python SDK. Anything you are able to share, would be great!

---

Yes, I'm using a Service Account to operate with the GCP. Pulumi invokes resources via Cloud SQL REST API under this account. That is why it should be enabled.

Great. I've just started. Will share what I got at some point